### PR TITLE
Code completed

### PR DESCRIPTION
**Pull request by mtalha-mergestack**
Ready for review

### CODE REVIEW

File: script.js
Comment (JavedJutt) [inline script.js:1]: why this file name is script.js?
what does it contain

File: style.css [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+body{
+    font-family: Arial, Helvetica, sans-serif; 
+}
+ a {
+      text-decoration: none;
+    }
+    .white-link {
+      color: white;
+      transition: color 0.1s;
+    }
```
Comment (JavedJutt) [inline style.css]: use proper formatting

File: style.css [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    .white-link4 :hover{
+        color: gray;
+    }
+    .white-link2:hover {
+        background-color: #3a8cc7;
+      }
+    .dropdown:hover>.dropdown-menu {
+        display: block;
+      }
```
Comment (JavedJutt) [inline style.css]: use proper names

File: index.html [hunk 3]
```diff
@@ -0,0 +1,645 @@
...
+        <div class="col-6 d-flex" style="height: 40px">
+          <img src="img/download.png" class="img-fluid my-1" style="height: 75%" />
+          <a
+            href="#"
+            class="white-link ms-3 my-1 mt-2"
+            style="
+              font-family: Arial, Helvetica, sans-serif;
+              font-size: 17.6px;
+              text-decoration: none;
+            "
```
Comment (JavedJutt) [inline index.html]: refactor this code and make components like Navbar for better understanding

File: index.html
Comment (JavedJutt) [inline index.html:1]: refactor this complete components make smaller chunks of components like header, navbar, etc

**Comment by JavedJutt** [comment]
@mtalha-mergestack refactor code and use proper names

File: script.js
Comment (mtalha-mergestack) [reply script.js:1]: It contains functions to handle vertical scroll  and zoom functionality

**Pull request merged**